# Set DC Motor Configuration Service
DCMotorConfig config
---
bool success
string message